src/modules/users/CheckPWQuality.h

```diff
@@ -38,10 +38,12 @@ class PasswordCheck
     using AcceptFunc = std::function<bool( const QString& )>;
     using MessageFunc = std::function<QString()>;
 
+    enum CheckId { DEFAULT, LIBPWQ };
+
     /** Generate a @p message if @p filter returns true */
-    PasswordCheck( MessageFunc message, AcceptFunc filter );
+    PasswordCheck( MessageFunc message, AcceptFunc filter, CheckId id = DEFAULT );
```

Comment:
Please don't use default values in constructors. If needed, add them explicitly where the constructors are called.